ecs-cli/modules/config/config.go

```diff
@@ -49,18 +52,26 @@ func NewCliConfig(cluster string) *CliConfig {
 }
 
 // ToServiceConfig creates an aws Config object from the CliConfig object.
-func (cfg *CliConfig) ToServiceConfig() (*aws.Config, error) {
+func (cfg *CliConfig) ToServiceConfig(providers ...credentials.Provider) (*aws.Config, error) {
```

Comment:
Since there aren't actually any callers that pass in `providers` right now (tests or otherwise), I'm in favour of not having the argument and then someone can add it if/when tests that need that mocking are added.